Extensions/TextObject/textruntimeobject.js

```diff
@@ -293,3 +325,121 @@ gdjs.TextRuntimeObject.prototype.setWrappingWidth = function(width) {
     this._wrappingWidth = width;
     this._renderer.updateStyle();
 };
+
+/**
+ * Set the outline for the text object.
+ * @param {String} str color as a "R;G;B" string, for example: "255;0;0"
+ * @param {number} thickness thickness of the outline (0 = disabled)
+ */
+gdjs.TextRuntimeObject.prototype.setOutline = function(str, thickness) {
+    var color = str.split(";");
+    if ( color.length < 3 ) return;
+
+    this._outlineColor[0] = parseInt(color[0], 10);
+    this._outlineColor[1] = parseInt(color[1], 10);
+    this._outlineColor[2] = parseInt(color[2], 10);
+    this._outlineThickness = thickness;
+    this._renderer.updateStyle();
+};
+
+/**
+ * Set the shadow for the text object.
+ * @param {String} str color as a "R;G;B" string, for example: "255;0;0"
+ * @param {number} distance distance between the shadow and the text
```

Comment:
Can you add the unit everytime it's needed?

`@param {number} distance distance between the shadow and the text, in pixels.`